modules/core/src/smithy4s/schema/SchemaVisitor.scala

```diff
@@ -29,8 +29,8 @@ trait SchemaVisitor[F[_]] extends (Schema ~> F) { self =>
   def union[U](shapeId: ShapeId, hints: Hints, alternatives: Vector[SchemaAlt[U, _]], dispatch: Alt.Dispatcher[Schema, U]): F[U]
   def biject[A, B](schema: Schema[A], bijection: Bijection[A, B]): F[B]
   def refine[A, B](schema: Schema[A], refinement: Refinement[A, B]): F[B]
-  def lazily[A](suspend: Lazy[Schema[A]]): F[A]
-  def nullable[A](schema: Schema[A]): F[Option[A]]
+  def lazily[A](shapeId: ShapeId, hints: Hints, suspend: Lazy[Schema[A]]): F[A]
```

Comment:
Nope, I'm veto-ing this change (same for nullable). If this change is required for the feature to work, I'll veto the feature.